### ISSUE TITLE
Link not copying to clipboard

### ISSUE THREAD

**Issue by fotoflo** [aviaryan/alfred-last2imgur#1]
Hi,
 Looks like a very promising workflow, but the link isn't copying to the clipboard thought it says it is, it's just clearing my clipboard actually.

 I'm using Alfred v3.4 - Am I doing anything wrong?

**Comment by aviaryan** [comment]
Hey,
do you see this dialog box when upload finishes? 
If yes, do you see the URL there?
![](https://i.imgur.com/4SIdyxS.png)

**Comment by fotoflo** [comment]
Yes, but without the URL.  Do i need to authenticate or anything to upload?

**Comment by aviaryan** [comment]
@fotoflo No, no auth is needed. And v3.4 is good as well. 
Btw, what is your screenshot folder? It should be `~/Desktop`.

**Comment by fotoflo** [comment]
hi @aviaryan where do i configure that?  I usually use "copy picture of selected area to the clipboard"

![image](https://user-images.githubusercontent.com/1642357/27677724-3e046f3c-5ce5-11e7-9e27-d160986f3848.png)

**Comment by aviaryan** [comment]
@fotoflo This extension uploads an image file to Imgur in the background. So it requires the screenshot to be saved in a file. So you will have to use (Shift-Command-4) to take a screenshot for this to work.

**Comment by fotoflo** [comment]
Hmm, alfred stores the "copy picture of selected area to the clipboard" images here: "copy picture of selected area to the clipboard" here: ~/Library/Application Support/Alfred 3/Databases/clipboard.alfdb.data... let me see if i can change your script

**Comment by fotoflo** [comment]
Yes, it worked!
Check the DIR line

```
#!/bin/bash

USER=$(whoami)
DIR="/Users/$USER/Library/Application Support/Alfred 3/Databases/clipboard.alfdb.data"
FILE=$(ls -t "$DIR" | head -n 1)

echo "$DIR/$FILE"
```

**Comment by aviaryan** [comment]
Fixed in v2.0.0
